releases/4.0.0-beta.2.md

```diff
@@ -0,0 +1,63 @@
+
+Release date: March 13, 2020
+
+TiDB version: 4.0.0-beta.2
+
+TiDB Ansible version: 4.0.0-beta.2
+
+## TiDB 4.0.0-beta.2 Compatibility Changes
+
++ Tools
+    - TiDB Binlog
+        - Fix the issue that the system returns an error and exits when `disable-dispatch` and `disable-causality` are configured in Drainer [#915](https://github.com/pingcap/tidb-binlog/pull/915)
+
+## New features
+
++ TiDB
+
++ TiKV
+    - Support the configuration of persistent dynamic update [#6684](https://github.com/tikv/tikv/pull/6684)
+
++ PD
+    - Support persisting the configuration of dynamic updates into hard drives [#2153](https://github.com/pingcap/pd/pull/2153)
+
++ Tools
+    - TiDB Binlog
+        - Support bi-directional data synchronization among TiDB clusters [#879](https://github.com/pingcap/tidb-binlog/pull/879)  [#903](https://github.com/pingcap/tidb-binlog/pull/903)
+    - TiDB Lightning
+        - Support configuring Transport Layer Security (TLS) [#40](https://github.com/tikv/importer/pull/40) [#270](https://github.com/pingcap/tidb-lightning/pull/270)
+    - TiCDC
+        - Initial release of the change data capture, providing the following features
+            - Support capturing change data from TiKV since v4.0.0-beta.2
+            - Support replicating change data to MySQL protocol-compatible database, with an eventual consistency guarantee
+            - Support replicating change data to Kafka, with either an eventual consistency guarantee or row-level order guarantee
+            - Provide process-level high availability
+    - BR
+        - Turn on experimental features such as incremental backup and saving backup files [#175](https://github.com/pingcap/br/pull/175)
+
++ TiDB Ansible
+    - Support injecting the information of related nodes to etcd [#1196](https://github.com/pingcap/tidb-ansible/pull/1196)
+    - Support deployingTiDB clusters on ARM [#1204](https://github.com/pingcap/tidb-ansible/pull/1204)
+
+## Bugfixes
+
++ TiDB
+
++ TiKV
+    - Fix the panic issue caused by empty short values during backup [#6718](https://github.com/tikv/tikv/pull/6718)
+    - Fix the issue that Hibernate Regions might not be woken up correctly [#6772](https://github.com/tikv/tikv/pull/6672) [#6648](https://github.com/tikv/tikv/pull/6648) [#6376](https://github.com/tikv/tikv/pull/6736)
+
++ PD
+    - Fix the panic issue that the rule checker fails to allocate Store to Region [#2160](https://github.com/pingcap/pd/pull/2160)
+    - Fix the issue that after the dynamic configuration is enabled, the configuration might have replication delay when the Leader is switched [#2154](https://github.com/pingcap/pd/pull/2154)
+
++ Tools
+    - BR
+        - Fix the issue that PD might fail to restore data because it cannot process large batches of data [#167](https://github.com/pingcap/br/pull/167)
+        - Fix the issue caused by upgrading TiDB when the BR version is not compatible with the TiDB version [#186]( https://github.com/pingcap/br/pull/186)
```

Comment:
I have checked with the PR author @overvenus that this is the correct meaning.
```suggestion
        - Fix the BR failure occurred because the BR version is not compatible with the TiDB version [#186](https://github.com/pingcap/br/pull/186)
```